src/main/game/CommandLine/CommandlineUi.java

```diff
@@ -39,12 +40,30 @@ public Moves getMove() {
         } catch (IOException e) {
             e.printStackTrace();
         }
-        if (userMove.equals("rock") || userMove.equals("paper") || userMove.equals("scissors")) {
-            return convertMove(userMove);
+        return getConvertedMove(userMove);
+    }
+
+    public Moves getMoveTwo(String gameMode) {
+        String humanMove = null;
+        Moves convertedMove;
+        if (gameMode.equals("2")) {
+            convertedMove = Computer.playMove();
         } else {
-            String translatedMove = translations.get(userMove);
-            return convertMove(translatedMove);
+            try {
+                askForMove();
+                humanMove = input.readLine();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+
+            if (language instanceof Greek) {
+                String translatedMove = getTranslatedMove(humanMove);
+                convertedMove = convertMove(translatedMove);
+            } else {
+                convertedMove = convertMove(humanMove);
+            }
         }
+        return convertedMove;
```

Comment:
I think this method has low cohesion as it has _multiple reasons to change_ that are _not related to each other_. For example:
1. It knows about game modes (human move vs computer move)
2. It knows about languages (english vs greek)

It's ok having one method that brings the above two ideas together like a wrapper, but the details of 1. and 2. can be in their own methods.

Reply:
The name of the method also makes me wonder... Why is there a `..Two` tacked at the end? 
Is it for the second move of a game? Why is it different from the first? 
Is it for _game mode 2_? Why does that need to be different from game mode 1?